src/common/registry/test/key-registry/registry-update.spec.ts

```diff
@@ -31,6 +31,8 @@ describe('Registry', () => {
   const CHAIN_ID = process.env.CHAIN_ID || 1;
   const address = REGISTRY_CONTRACT_ADDRESSES[CHAIN_ID];
 
+  const blockHash = '0x4ef0f15a8a04a97f60a9f76ba83d27bcf98dac9635685cd05fe1d78bd6e93418';
```

Comment:
We don't use ValidatorRegistryService in kapi , but covered it with tests more than KeyRegistryService.update. I see they are similar the only diff is toIndex, but maybe better to add those test here too

Reply:
added a similar test for another module